airflow/dags/amplitude_benefits_views/METADATA.yml

```diff
@@ -0,0 +1,6 @@
+description: "Load events from BigQuery to view"
+schedule_interval: "0 0 * * *"
+catchup: True
```

Comment:
**suggestion**: I think we set `catchup` to `False` here.

Since this DAG simply copies over whatever is in the `amplitude.benefits_events` table, it doesn't need to backfill - one run for the latest date is sufficient.